src/clip.sol

```diff
@@ -210,7 +210,7 @@ contract Clipper {
     // Could get this from rmul(Vat.ilks(ilk).spot, Spotter.mat()) instead, but
     // if mat has changed since the last poke, the resulting value will be
     // incorrect.
-    function getPrice() internal returns (uint256 price) {
+    function getFeedPrice() internal returns (uint256 price) {
```

Comment:
Might as well name the return var `feedPrice`.

Reply:
done!